scss/common/_animation.scss

```diff
@@ -70,6 +70,7 @@ Enable to expend a folded block element
 .sdc-ripple-click__animated::before{
   display: inline-block;
   position:absolute;
+  top:0;
   content: '';
   animation: ripple-animation .3s ease-out; 
   background-color: #009fdb;
```

Comment:
Why is z-index being used here? If the animation looks good without it, or if you can get the behavior right using 'order' instead, then please remove z-index

Reply:
Agreed, there's no need for z-index here ('order' isnt related though).